### PR TITLE
4399 dev apply again content changes

### PR DESCRIPTION
**Pull request by avinhurry**
## Context
After a content audit, we are making these changes to emails and content in line with the new Apply again with three choices.

## Changes proposed in this pull request

- Update email content
- Reuse the same template that we use for Apply1 if the feature flag is on
- Duplicated the test for ease of removal
- Update terms and conditions in line with the three choices change
- Remove unused email (this was previously used as a one off according to the comment in the file)

## Guidance to review

 - Check the content
 - Does the logic to render the apply1 email if the feature flag is active look ok?
 - Are we sure the email removed is no longer required?

## Link to Trello card

https://trello.com/c/e8tLDNVg/4399-dev-apply-again-content-changes

### CODE REVIEW

**Comment by EmmaFrith** [comment]
In the application_rejected_all_applications_rejected_after_apply2_deadline email, can we change:

'Your last application has been saved. Make any changes and re-submit from 6:28pm on 23 February 2022.'

To:

Your last application is saved. All you need to do is:

- make any changes
- choose up to 3 courses
- submit

I must've missed this one from the audit @avinhurry

File: app/views/candidate_mailer/_still_interested_content.text.erb [hunk 1]
```diff
@@ -8,7 +8,12 @@ Your last application has been saved. Make any changes and re-submit from <%= Cy
 
 If now’s the right time for you, you can still apply for teacher training again this year.
 
-Your last application has been saved. You can make changes and choose a different course before you submit a new application.
+Your last application is saved. All you need to do is:
+
+  - Make any changes
+  - Choose up to 3 courses
```
Comment (EmmaFrith) [inline app/views/candidate_mailer/_still_interested_content.text.erb:14]:
```suggestion
  - choose up to 3 courses
```

File: app/views/candidate_mailer/_still_interested_content.text.erb [hunk 2]
```diff
@@ -8,7 +8,12 @@ Your last application has been saved. Make any changes and re-submit from <%= Cy
...
+  - Submit
```
Comment (EmmaFrith) [inline app/views/candidate_mailer/_still_interested_content.text.erb:15]:
```suggestion
  - submit
```

**Comment by EmmaFrith** [comment]
On the 'application_submitted_apply_again' mailer, does 'training provider' pluralise if they submit more than one application?: 

'If your training provider decides to progress your application, they’ll contact you to organise an interview.'

If there's more than one course, are the courses listed in bullet points as in the Apply 1 email?

**Comment by EmmaFrith** [comment]
are we doing the terms of use changes in this PR too @avinhurry?

**Review by EmmaFrith — CHANGES_REQUESTED** [review]
Thanks @avinhurry, it's looking great. I've added some questions and suggestions.

**Comment by avinhurry** [comment]
> Your last application is saved. All you need to do is:
> 
>     * make any changes
> 
>     * choose up to 3 courses
> 
>     * submit from [time] on [date]

This was changed, it just doesn't show in the preview because there's a conditional in the email template. It has to be between a certain time period for this to show. I'll attach a screenshot here. It just doesn't have the time and the date on the third bullet... 

image

File: app/views/candidate_mailer/_still_interested_content.text.erb [hunk 3]
```diff
@@ -8,7 +8,12 @@ Your last application has been saved. Make any changes and re-submit from <%= Cy
 
 If now’s the right time for you, you can still apply for teacher training again this year.
 
-Your last application has been saved. You can make changes and choose a different course before you submit a new application.
+Your last application is saved. All you need to do is:
+
+  - Make any changes
```
[the remark this answers is not in the document]
Comment (avinhurry) [reply app/views/candidate_mailer/_still_interested_content.text.erb:13]: oops, I did ask about this and you did say, sorry 😞

Changes made

File: app/views/candidate_mailer/_still_interested_content.text.erb (hunk 1 above)
Comment (avinhurry) [reply app/views/candidate_mailer/_still_interested_content.text.erb:14]: same here

File: app/views/candidate_mailer/_still_interested_content.text.erb (hunk 2 above)
Comment (avinhurry) [reply app/views/candidate_mailer/_still_interested_content.text.erb:15]: and here

**Comment by avinhurry** [comment]
> On the 'application_submitted_apply_again' mailer, does 'training provider' pluralise if they submit more than one application?:
> 
> 'If your training provider decides to progress your application, they’ll contact you to organise an interview.'
> 
> If there's more than one course, are the courses listed in bullet points as in the Apply 1 email? @avinhurry

If the `apply_again_with_three_choices` feature flag is active, the `application_submitted` email is sent, once we delete the feature flag we will delete the `application_submitted_apply_again` email template because it won't be used.

**Comment by avinhurry** [comment]
> are we doing the terms of use changes in this PR too @avinhurry?

You mean this file, right? `terms_candidate.md` Yes i've updated it - it's in this PR.

**Comment by EmmaFrith** [comment]
@avinhurry could you change this sentence

Screenshot 2022-02-17 at 10 14 40

to:

You can apply **again** if your initial application does not lead to a place.

**Review by stevehook — APPROVED** [review]
This all makes sense to me, good stuff! I can't see any references to the `apply_again_call_to_action` email. And it was a one-off so there shouldn't be any jobs for it left in the Sidekiq queue.

**Review by EmmaFrith — APPROVED** [review]
LGTM. Thanks Avin!

**Pull request merged**